tests/docker-integration/Process/Bash.php

```diff
@@ -35,11 +33,16 @@ public function createCompose(string $command, string $container, array $variabl
             $command
         );
 
-        return $this->create($command);
+        parent::__construct($command);
     }
 
-    private function getDefaultEnvironmentVariables(): array
+    /**
+     * @return array
+     */
+    private function getDefaultVariables(): array
     {
+        $config = new Config();
+
         return [
             'MAGENTO_CLOUD_RELATIONSHIPS' => [
                 'database' => [
```

Comment:
I see these credentials in 3 places. We may think how to move them in one common place to manage.